### ISSUE TITLE
[pattern library] [dropdown menu] only the first item should be in the tab index

### ISSUE THREAD

**Issue by ryanflorence** [a11yproject/a11yproject.com#248]
According to WAI-ARIA specs:

> First item in menu bar should be in the tab order (tabindex=0)

http://www.w3.org/TR/wai-aria-practices/#menu

The pattern implementation has every item in the menu tabbable. This need not be the case, once the user tabs to the menu, they then use the arrow keys to navigate it.

The WAI ARIA document doesn't fully explain the navigation, however. In voiceover, if you are navigating the OS X application menu (get there with VO + M):
- tab moves across the top level menus, arrows navigate the dropdown items
- menus do not open on focus (this allows you to tab from one menu to another, instead of down every list)

I would personally implement it this way since the HTML control is emulating it.

**Comment by svinkle** [comment]
This issue has become "[stale](https://github.com/a11yproject/a11yproject.com/blob/gh-pages/CONTRIBUTING.md#stale-issues)" — @ryanflorence, feel free to open a [new issue](https://github.com/a11yproject/a11yproject.com/issues/new) if you'd like to continue.
